.cypress/integration/trace_analytics_test/trace_analytics_dashboard.spec.js

```diff
@@ -250,25 +250,22 @@ describe('Testing filters on trace analytics page', { scrollBehavior: false }, (
   });
 
   it('Verify Change all filters', () =>{
-    cy.wait(delay);//Needed after removing waits from setTimeFilter()
     cy.get('[data-test-subj="global-filter-button"]').click();
     cy.get('.euiContextMenuPanelTitle').contains('Change all filters').should('exist');
-    cy.get('.euiContextMenuItem__text').eq(0).contains('Enable all');
-    cy.get('.euiContextMenuItem__text').eq(1).contains('Disable all');
-    cy.get('.euiContextMenuItem__text').eq(2).contains('Invert inclusion');
-    cy.get('.euiContextMenuItem__text').eq(3).contains('Invert enabled/disabled');
-    cy.get('.euiContextMenuItem__text').eq(4).contains('Remove all');
+    cy.get('.euiContextMenuItem__text').eq(1).contains('Enable all');
+    cy.get('.euiContextMenuItem__text').eq(2).contains('Disable all');
+    cy.get('.euiContextMenuItem__text').eq(3).contains('Invert inclusion');
+    cy.get('.euiContextMenuItem__text').eq(4).contains('Invert enabled/disabled');
+    cy.get('.euiContextMenuItem__text').eq(5).contains('Remove all');
```

Comment:
Not a blocker, however, I was wondering if we can do something like this:

```javascript
const menuSelector = '.euiContextMenuItem__text';
const expectedMenuItems = [
  'Enable all',
  'Disable all',
  'Invert inclusion',
  'Invert enabled/disabled',
  'Remove all',
];

cy.verifyContextMenuItems(menuSelector, expectedMenuItems);
```

Reply:
It looks like cypress can do something like that by iterating through each menu item to check but I think it might be more clear to have the order of the menu items laid out so holding off on this for now